project-management/Github.md

```diff
@@ -44,6 +44,18 @@ the milestone description to make the meaning clear.
 
 [github-milestones]: https://help.github.com/articles/about-milestones/
 
+## Epics
+
+[Epics][atlassian-epic] map to ["projects"][github-projects]. All the
```

Comment:
I don't think starting the link to epic is very helpful. I don't imagine more than half of the potential readers of that guide know what they are. (the atlassian link isn't super helpful either). I'd only put it in a side-note (for those who already are familiar with the concept).

Instead, we should try and delineate when to reach for a project. Attempt (disjunctive bullet point list):

- When you have several semantic group of tasks.
- When a group of task spans over more than one milestone.
- When task management becomes unwieldy, even if you have just the one.

(it's not great, but it's a start)

Reply:
Clients sometimes ask about epics (which is why I'm adding this section). It's important to be able to pattern match on the client's specific ask when that arises. Also, bias note: anyone who's done Scrum at least once would normally know what an epic is (epic, user story, task are the three main levels of organization in Agile). Each item in your disjunction actually describe one aspect of what an epic is, so better to refer to the already well established vocabulary for this, rather than inventing our own bespoke semantics. If you can find a better resource than the Atlassian link (which incidentally, includes all three points in your disjunction), feel free to propose another one.

Reply:
I guess what it comes down to is: what is this guide's main audience? If it's clients trying to understand how we can collaborate, then I'd say that this section is about as good as it can get. But if, as I was assuming, the audience is Tweag engineers, especially people who joined recently, who want to learn about our ways, then I still feel it's not very helpful. The Atlassian link is pretty good, but it's longer than the entire guide. I don't think it is reasonable to expect that most hirees have been through a scrum process before.

I don't mean remove epics from the section: it's helpful if you already know what they are. But I wouldn't make it the starting point. I'd do something like:

> We reach for project where <…>. We use projects thus <…>. Projects roughly correspond to [epics][atlassian-epic]  in agile workflows